doc/sphinx/language/core/inductive.rst

```diff
@@ -3,6 +3,22 @@ Inductive types and recursive functions
 
 .. _gallina-inductive-definitions:
```

Comment:
This label should stay right before the corresponding header. This is what makes the build currently fail.